src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b)) {
+    let count;
+
```

Comment:
The variable 'count' is not descriptively named. Consider renaming it to something more descriptive like 'didSwap' or 'isSorted'.